library/core/src/sync/atomic.rs

```diff
@@ -1179,6 +1200,14 @@ impl<T> AtomicPtr<T> {
                 Err(x) => Err(x as *mut T),
             }
         }
+        #[cfg(not(bootstrap))]
+        // SAFETY: This intrinsic is unsafe because it operates on a raw pointer
+        // but we know for sure that the pointer is valid (we just got it from
+        // an `UnsafeCell` that we have by reference) and the atomic operation
+        // itself allows us to safely mutate the `UnsafeCell` contents.
```

Comment:
This doesn't match the safety comment for the `#[cfg(bootstrap)]` case.

Reply:
I know, but I felt like it wasn't worth updating that, too, since it'll go away with the next beta anyway